red_blue_world/Quilt.py

```diff
@@ -92,12 +98,15 @@ def _maybe_unload(self, patch_id: PatchID) -> None:
 
         # assumes that `self._unload_clocks` is usually empty or very small
         # otherwise, this would cause a performance degradation on every tick
+        delete_lst = []
```

Comment:
Instead of keeping this list, it would be better to add this to `unload_patch` below. For example:
```python
    def unload_patch(self, patch_id: PatchID) -> None:
        # this should call out to the patch_loader
        # probably this can be inlined. Keeping as a type-stub for now
        del self._patches[patch_id]
        del self._unload_clocks[patch_id]
```